### ISSUE TITLE
i have already installed (segment anything) but following node types were not found while using

### ISSUE THREAD

**Issue by Manish-cel** [storyicon/comfyui_segment_anything#12]
When loading the graph, the following node types were not found:
InvertMask (segment anything)
GroundingDinoSAMSegment (segment anything)
GroundingDinoModelLoader (segment anything)
SAMModelLoader (segment anything)
Nodes that have failed to load will show as red on the graph.

i have already installed (segment anything) and I'm still getting this error. please help

**Comment by GoDJr** [comment]
I’m having the same problem suddenly

**Comment by txs** [comment]
I'm having this issue too! What's happening?

**Comment by GoDJr** [comment]
Use update all in comfyui manager. I think some of the other addons that provides dependencies for segment all needed to be updated. I did update all today and it fixed my problem but now I have the type error: cannot unpack non-iterable NoneType object issue.

**Comment by txs** [comment]
> Use update all in comfyui manager. I think some of the other addons that provides dependencies for segment all needed to be updated. I did update all today and it fixed my problem but now I have the type error: cannot unpack non-iterable NoneType object issue.

Thanks for the quick reply! I'll try your method and hopefully it would work!

**Comment by txs** [comment]
> Use update all in comfyui manager. I think some of the other addons that provides dependencies for segment all needed to be updated. I did update all today and it fixed my problem but now I have the type error: cannot unpack non-iterable NoneType object issue.

I update all but I still cannot find the "groundingDino".
I'm using it in the portable version of the ComfyUI. How do I fix this.

Is it possible for you to zip your folder and I can download it to test it out? Thanks!

**Comment by Manish-cel** [comment]
> Use update all in comfyui manager. I think some of the other addons that provides dependencies for segment all needed to be updated. I did update all today and it fixed my problem but now I have the type error: cannot unpack non-iterable NoneType object issue.

already did that still giving that error

**Comment by txs** [comment]
Hi guys, I kind of know what’s the issue. That is when the groundDino cannot find the prompt. So that the images and masks can be wrong.


The next issue I’m having is that torch.cat cannot process with empty list of res_images, and res_masks. How do I create empty image or mask to return for the main function?

**Comment by ManuelMultiverse** [comment]
Can´t find the nodes.

**Comment by wastelandlab** [comment]
I have the same issue. Unable to find "SAMModelLoader (segment anything)", "GroundingDinoSAMSegment (segment anything)" and "GroundingDinoModelLoader (segment anything)"

"update all in comfyui manager" didn't do anything 

Anyone any news on this?

**Comment by univevoca** [comment]
I still have the issue on runpod. Even after installing the missing requirements. On rundiffuson it works. But that really does not help me.
Anyone any news?

**Comment by storyicon** [comment]
Now dependencies will be installed automatically, and this issue should have been resolved. If you have any other problem, feel free to reopen the issue.
